src/api/built-in-directives.md

```diff
@@ -319,13 +319,16 @@ Dynamically bind one or more attributes, or a component prop to an expression.
   <svg><a :xlink:special="foo"></a></svg>
   ```
 
-  The `.prop` modifier also has a dedicated shorthand, `.`:
+  The `.prop` and `.attr` modifiers also have a dedicated shorthand, `.` and `^` respectively:
 
   ```vue-html
   <div :someProperty.prop="someObject"></div>
-
   <!-- equivalent to -->
   <div .someProperty="someObject"></div>
+  
+  <div :someProperty.attr="someObject"></div>
+  <!-- equivalent to -->
+  <div ^someProperty="someObject"></div>
```

Comment:
I'm not exactly sure of the typical use case, but would it be more accurate for the example to use `someString` rather than `someObject` for the `.attr`/`^` example? I assume it would need to be a string to be set as an attribute?

```suggestion
  <div :someProperty.attr="someString"></div>
  <!-- equivalent to -->
  <div ^someProperty="someString"></div>
```